Imagine/Filter/FilterManager.php

```diff
@@ -160,7 +160,16 @@ private function exportConfiguredImageBinary(BinaryInterface $binary, ImageInter
         }
 
         $filteredFormat = $config['format'] ?? $binary->getFormat();
-        $filteredString = $image->get($filteredFormat, $options);
+        try {
+            $filteredString = $image->get($filteredFormat, $options);
+        } catch (\Exception $exception) {
+            if ('webp' === $filteredFormat && true === $options['animated']) {
```

Comment:
i think we need to check if the option is set as well:
```suggestion
            if ('webp' === $filteredFormat && array_key_exists('animated', $options) && true === $options['animated']) {
```